Cargo.toml

```diff
@@ -104,7 +104,7 @@ features = ["secure"]
 [profile.dev]
 opt-level = 0  # Controls the --opt-level the compiler builds with
 debug = true   # Controls whether the compiler passes `-g`
-codegen-units = 2
+codegen-units = 4
```

Comment:
why use 4 here?

Reply:
Both of them are trying to fix compilation failures in the Travis. See more https://github.com/pingcap/tikv/pull/1764

Reply:
oh, so you revert the change?

Reply:
Yes, tikv is no longer tested on the Travis anyway.

@BusyJay Do you have any suggestions?

Reply:
Looks OK to me.